pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -96,8 +96,8 @@ func NewKubeGenericRuntimeManager(
 	cpuCFSQuota bool,
 	runtimeService internalApi.RuntimeService,
 	imageService internalApi.ImageManagerService,
-) (*kubeGenericRuntimeManager, error) {
-	kubeRuntimeManager := &kubeGenericRuntimeManager{
+) (*KubeGenericRuntimeManager, error) {
```

Comment:
@yujuhong This is for reusing `NewKubeGenericRuntimeManager` in testing.  If not export, then we should write another `NewKubeGenericRuntimeManager` for testing, which should returns `KubeGenericRuntimeManager` instead of `kuberuntime.Runtime`.